.github/workflows/pull.yml

```diff
@@ -146,7 +146,13 @@ jobs:
       docker-image-name: pytorch-linux-bionic-py3.11-clang9
       test-matrix: |
         { include: [
-          { config: "smoke", shard: 1, num_shards: 1, runner: "linux.2xlarge" },
+          { config: "default", shard: 1, num_shards: 2, runner: "linux.2xlarge" },
+          { config: "default", shard: 2, num_shards: 2, runner: "linux.2xlarge" },
+          { config: "crossref", shard: 1, num_shards: 2, runner: "linux.2xlarge" },
+          { config: "crossref", shard: 2, num_shards: 2, runner: "linux.2xlarge" },
+          { config: "dynamo", shard: 1, num_shards: 2, runner: "linux.2xlarge" },
+          { config: "dynamo", shard: 2, num_shards: 2, runner: "linux.2xlarge" },
```

Comment:
This would fail, wouldn't it?

Reply:
Yes, I want to see how this looks right now, won't merge yet.

Reply:
btw all dynamo tests are already skipped today. So this should work as is. It will make it easier for William to enable these tests if the shards are already there.